src/utils.rs

```diff
@@ -21,17 +22,12 @@ pub const PARSER_CONFIG: ParserConfig = ParserConfig {
 };
 
 /// Prints an error to `stderr` in red.
-pub fn print_error<S: AsRef<str>>(error: S, verbose: bool) {
+pub fn print_error<S: AsRef<str>>(error: S, _: bool) {
     if cfg!(not(test)) {
-        let _ =
-            io::stderr()
-                .write(&format!("{} {}\n", "Error:".bold().red(), error.as_ref().red())
-                    .into_bytes()[..])
-                .unwrap();
-
-        if !verbose {
-            println!("If you need more information, try to run the program again with the {} \
-                      flag.",
+        error!("{}", error.as_ref());
```

Comment:
Is this using `stderr`? That's a big deal, since we rely on printing using `stderr` for everything (except debug and some prints in non-quiet mode).